catalog/OpenET/OpenET_GEESEBAL_CONUS_GRIDMET_MONTHLY_v2_0.jsonnet

```diff
@@ -0,0 +1,164 @@
+    algorithms developed by Bastiaanssen et al. (1998). The OpenET geeSEBAL implementation uses land surface
+    temperature (LST) data from Landsat Collection 2, in addition to NLDAS and gridMET datasets as instantaneous
+    and daily meteorological inputs, respectively. The automated statistical algorithm to select the hot and
+    cold endmembers is based on a simplified version of the Calibration using Inverse Modeling at
+    Extreme Conditions (CIMEC) algorithm proposed by Allen et al. (2013), where quantiles of LST and the
+    normalized difference vegetation index (NDVI) values are used to select endmember candidates in the
+    Landsat domain area. The cold and wet endmember candidates are selected in well vegetated areas, while
+    the hot and dry endmember candidates are selected in the least vegetated cropland areas. Based on the
+    selected endmembers, geeSEBAL assumes that in the cold and wet endmember all available energy is converted
+    to latent heat (with high rates of transpiration), while in the hot and dry endmember all available energy is
+    converted to sensible heat. Finally, estimates of daily evapotranspiration are upscaled from instantaneous
+    estimates based on the evaporative fraction, assuming it is constant during the daytime without significant
+    changes in soil moisture and advection. Based on the results from the OpenET Accuracy Assessment and
+    Intercomparison study, the OpenET geeSEBAL algorithm was modified as follows: (i) the simplified version of
+    CIMEC was improved by using additional filters to select the endmembers, including the use of the
+    USDA Cropland Data Layer (CDL) and filters for NDVI, LST and albedo; (ii) corrections to LST for
+    endmembers based on antecedent precipitation; (iii) definition of NLDAS wind speed thresholds to reduce
+    model instability during the atmospheric correction; and, (iv) improvements to estimate daily net
+    radiation, using FAO-56 as reference (Allen et al., 1998). Overall, geeSEBAL performance is dependent on
+    topographic, climate, and meteorological conditions, with higher sensitivity and uncertainty related to hot
+    and cold endmember selections for the CIMEC automated calibration, and lower sensitivity and uncertainty
+    related to meteorological inputs (Laipelt et al., 2021 and Kayser et al., 2022). To reduce uncertainties
+    related to complex terrain, improvements were added to correct LST and global (incident) radiation on the
+    surface (including the environmental lapse rate, elevation slope and aspect) to represent the effects of
+    topographic features on the model’s endmember selection algorithm and ET estimates.
+
+    [Additional information](https://openetdata.org/methodologies/)
+  |||,
+  license: license.id,
+  links: ee.standardLinks(subdir, id),
+  keywords: [
+    'conus',
+    'evapotranspiration',
+    'gridmet_derived',
+    'landsat_derived',
+    'monthly',
+    'water',
+    'geesebal'
```

Comment:
We are trying to avoid adding new keywords at this point and this one is only used in this file. Please comment out the keyword and we can consider adding it later.  Dataset search will find it as geeSEBAL is in the description text and the title which are both included in the text to search.